components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/valve/TenantContextRewriteValve.java

```diff
@@ -41,12 +41,13 @@
 import java.util.List;
 import java.util.Map;
 import java.util.regex.Pattern;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletResponse;
 
 public class TenantContextRewriteValve extends ValveBase {
 
-    private static final String TENANT_NAME_FROM_CONTEXT = "TenantNameFromContext";
+    public static final String TENANT_NAME_FROM_CONTEXT = "TenantNameFromContext";
```

Comment:
Should we have this. Isn't this defined in 'IdentityCoreConstants' to be used across components and this respective module has dependency for core